test/integration/controller/admissionchecks/provisioning/provisioning_test.go

```diff
@@ -40,7 +42,7 @@ const (
 	customResourceTwo = "example.org/res2"
 )
 
-var _ = ginkgo.Describe("Provisioning", func() {
+var _ = ginkgo.Describe("Provisioning", ginkgo.Ordered, ginkgo.ContinueOnFailure, func() {
```

Comment:
why did this change?

Reply:
I modify the `vars`, and different tests use different value, so I don't want the tests to run in parallel, but maybe this is not needed?